rest/replicatortest/replicator_test_legacy_rev_test.go

```diff
@@ -826,28 +826,28 @@ func TestActiveReplicatorConflictPreUpgradedVersionOneSide(t *testing.T) {
+		// +-----------------+-------------------+--------------------------------------+-------------------+
+		// |                 | SGW1              |                  | SGW2              |                   |
+		// +-----------------+-------------------+------------------+-------------------+-------------------+
+		// |                 | Rev Tree          | HLV              | Rev Tree          | HLV               |
+		// +-----------------+-------------------+------------------+-------------------+-------------------+
+		// | Initial State   | 2-def,1-abc       | 100@SGW1         | 2-abc,1-abc       | none              |
+		// +-----------------+-------------------+------------------+-------------------+-------------------+
+		// | Expected Result | 3-abc,2-abc,1-abc | 3abc@RTE;100@SGW | 3-abc,2-abc,1-abc | 3abc@RTE;100@SGW1 |
+		// +-----------------+-------------------+--------------------------------------+-------------------+
 		{
 			name:                            "active peer has post upgrade version that wins",
 			activePeerHasPostUpgradeVersion: true,
 		},
-		// +-----------------+-------------+----------+-------------+----------+--+--+--+--+--+
-		// |                 | SGW1        |          | SGW2        |          |  |  |  |  |  |
-		// +-----------------+-------------+----------+-------------+----------+--+--+--+--+--+
-		// |                 | Rev Tree    | HLV      | Rev Tree    | HLV      |  |  |  |  |  |
-		// +-----------------+-------------+----------+-------------+----------+--+--+--+--+--+
-		// | Initial State   | 2-abc,1-abc | none     | 2-def,1-abc | 100@SGW2 |  |  |  |  |  |
-		// +-----------------+-------------+----------+-------------+----------+--+--+--+--+--+
-		// | Expected Result | 2-def,1-abc | 100@SGW2 | 2-def,1-abc | 100@SGW2 |  |  |  |  |  |
-		// +-----------------+-------------+----------+-------------+----------+--+--+--+--+--+
+		// +-----------------+-------------+-------------------------------+-------------+--------------------------------+
+		// |                 | SGW1        |                               | SGW2        |                                |
+		// +-----------------+-------------+-------------------------------+-------------+--------------------------------+
+		// |                 | Rev Tree    | HLV                           | Rev Tree    | HLV                            |
+		// +-----------------+-------------+-------------------------------+-------------+--------------------------------+
+		// | Initial State   | 2-abc,1-abc | none                          | 2-def,1-abc | 100@SGW2                       |
+		// +-----------------+-------------+-------------------------------+-------------+--------------------------------+
+		// | Expected Result | 2-def,1-abc | 1100@SGW2;2def@RTE;oldcas@SGW1| 2-def,1-abc | 1100@SGW2;2def@RTE;oldcas@SGW1 |
```

Comment:
I think here we need to swap the sources for SGW1 and SGW2 given for this test case we update the doc on active peer (SGW1) on line 1005. So I think this needs to be 1100@SGW1;2def@RTE,oldcas@SGW2